wikimedia/wikipedia-ios
# EditAttemptStep - adjust payload to fix POST errors

**Phabricator:**
https://phabricator.wikimedia.org/T313561

### Notes
Our POST calls were failing in our new EditAttemptStep schema. This PR adjusts the payload to match what Android was sending. We went from this:

```
{
  "app_install_id": "04985969-5E6E-4E98-A482-CA5445394736",
  "dt": "2023-05-24T20:39:53Z",
  "editing_session_id": "",
  "version": 1,
  "editor_interface": "wikitext",
  "primary_language": "test",
  "is_anon": true,
  "user_id": 0,
  "action": "init",
  "app_session_id": "7b672d3277ef14bd5e16",
  "platform": "phone",
  "meta": {
    "stream": "eventlogging_EditAttemptStep",
    "id": "0F49BC0A-8921-41A6-8D8E-B5FEFB3CC8BB"
  },
  "page_title": "Hello World",
  "user_editcount": 0,
  "page_ns": 0,
  "integration": "app-ios",
  "mw_version": "",
  "$schema": "/analytics/legacy/editattemptstep/1.4.0"
}
```

To this:

```
{
  "meta": {
    "stream": "eventlogging_EditAttemptStep",
    "id": "430C915D-C1A7-49C7-ADA2-36D493BD9D0C"
  },
  "event": {
    "action": "init",
    "page_title": "Hello World",
    "editor_interface": "wikitext",
    "page_ns": 0,
    "platform": "phone",
    "integration": "app-ios",
    "user_editcount": 0,
    "editing_session_id": "",
    "user_id": 0,
    "version": 1,
    "mw_version": ""
  },
  "dt": "2023-05-24T21:53:36Z",
  "$schema": "/analytics/legacy/editattemptstep/1.4.0"
}
```

Changes made were removing app_install_id, app_session_id, is_anon, and primary_language, and nesting the event data into an "event" key.

### Test Steps
1. Click an article edit pencil, confirm an event POST for the `init` action goes through without an error. There should be no need to test the other actions.
